test/e2e/source_apiserver_test.go

```diff
@@ -29,7 +29,10 @@ func TestSourceApiServer(t *testing.T) {
 	t.Parallel()
 	test := NewE2eTest(t)
 	test.Setup(t)
-	defer test.Teardown(t)
+	defer func() {
+		test.deleteServiceAccountForApiserver(t, "testsa")
```

Comment:
Names for ServiceAccount, ClusterRole and ClusterRoleBindings can be set as constants and used uniformly for create/delete and references.